packages/emails/templates/attendee-scheduled-email.ts

```diff
@@ -23,7 +23,7 @@ export default class AttendeeScheduledEmail extends BaseEmail {
     this.showAttendees = showAttendees;
     this.t = attendee.language.translate;
 
-    if (!this.showAttendees) {
+    if (!this.showAttendees && this.calEvent.seatsPerSlotTimeSlot) {
```

Comment:
Since `showAttendees` is default false, we only want to hide attendees when there are seats with the event type.